### DISCUSSION TITLE
Disabling LLM-Generated Titles by default to reduce token consumption

### DISCUSSION THREAD

**Discussion by Genai-labs** [open-webui/open-webui#5167] [Ideas]
Currently, the title generation process is powered by a Large Language Model (LLM) and is enabled by default. While this feature can enhance user experience by providing contextually relevant titles, it also consumes tokens during each activation, which can lead to unnecessary usage of resources.

To optimize token consumption and reduce costs, it would be beneficial to have this functionality disabled by default. This would ensure that the model is only solicited when explicitly needed, preventing unintentional consumption of tokens. Administrators could have the option to manually activate the feature whenever LLM-generated titles are required, allowing for better control over resource utilization.

**Comment by spammenotinoz** [comment]
Use a cheap model (defined in settings), even under high load the cost wouldn't even be in cents.

**Reply by Levara** [reply]
Cheap models write dumb titles for non-english threads. So in that case the feature not only doesn't improve user experience, it worsens it. Especially since it's rerunning on each regeneration, which means that if you are testing the output of multiple models, it will rewrite the title, even if you rename the thread manually.

It would be great if there could be an option to disable this, it doesn't even have to be disabled by default.

**Comment by mal1k-me** [comment]
I totally agree. Running a model via `ollama run <model_name>` for example is less heavy on resources compared to running it via Open WebUI with all extra features disabled.  
Even though choosing a modest model for chat titles generation is a viable option, disabling the feature itself remains a good tweak for lower end machines and laptops with mobile chips.
The user can be prompted to manually provide a title and/or tags before closing the session.
